### PR TITLE
ReworkDeck

### PR DESCRIPTION
**Pull request by studiotbt**
1. note area larger. 2. larger note click area. 3. close btn clearer. 4. removed bottom line (!! :-)).

### CODE REVIEW

File: button/popup.css [hunk 1]
```diff
@@ -56,25 +56,29 @@ http://matthewlein.com/ceaser/
...
     background: #fff;
+    float: left;
     cursor: default;
     border-top: 1px solid #e3e3e3;
 }
 
 .site { /* each note area */
     height: 50px; /* prev hieght 40px */
-    padding: 2px 0 2px 0;
+    margin: 2px 0 0px 0;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/1772214731/issues/source?bid=2884821&fileBranchId=3065795#l70)

File: button/popup.css [hunk 2]
```diff
@@ -56,25 +56,29 @@ http://matthewlein.com/ceaser/
...
     border-top: 1px solid #e3e3e3;
     border-bottom: 1px solid #e3e3e3;
     position: relative;
     z-index: 1;
     width: 100%;
     cursor: pointer;
+}
 
+.site:last-child {
+    border-bottom:0px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/1772214731/issues/source?bid=2884821&fileBranchId=3065795#l80)

File: button/popup.css [hunk 3]
```diff
@@ -292,11 +296,11 @@ http://matthewlein.com/ceaser/
 }
 
 .sitetitle {
-    margin-top: 3px;
     display: block;
-    left: -10px;
+    height: 35px;
     position: inherit;
-    padding: 10px 0 0 15px;
+    padding: 15px 10px 0 5px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Using height with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/tomastaro/notemark/file/1772214731/issues/source?bid=2884821&fileBranchId=3065795#l302)

**Comment by studiotbt** [comment]
corrected the codacy css issues. :-)

**Comment by tomasko126** [comment]
@studiotbt I will close this one in favor of #71.

**Pull request closed without merging**